### PR TITLE
Tests for formato_json.py module

### PR DESCRIPTION
**Pull request by ABHISHEKSONI121**
# Test Suite Summary for `formato_json` Module

### Key Achievements

- 14 tests passing
- Comprehensive testing for `leer` and `escribir` functions
- Covers JSON file reading, writing, and data manipulation
- Includes edge cases and exception handling

## Manual test evidence

finalllll

## Checklist

- [x] Classes, Variables, function and methods logic ok
- [x] Comments written explaining what the code does
- [x] All python code is PEP8 compliant (run black .)
- [x] No lint issues (run flake8)
- [x] Test coverage with pytest implemented
- [x] Reviewers assigned (at least 1 mentor)

### 🔗 Related Issues

Closes #164: Improve test coverage for `formato_json` module

---

> 💡 This test suite provides excellent coverage for the `formato_json` module. It demonstrates the reliability and robustness of the JSON processing functionality in PyAfipWs.

### CODE REVIEW

File: tests/test_formato_json.py [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+import unittest
+import json
+import stat
+import pytest
+from decimal import Decimal
+from pyafipws.formatos.formato_json import leer, escribir
+import tempfile
+
+@pytest.mark.dontusefix
+class TestFormatoJSON(unittest.TestCase):
```
Comment (NicolasSandoval) [inline tests/test_formato_json.py]: @ABHISHEKSONI121 Could you write the test with pytest? You could use the pytest `tmp_path` fixture for this.

**Review by NicolasSandoval — CHANGES_REQUESTED** [review]

File: tests/test_formato_json.py (hunk 1 above)
Comment (ABHISHEKSONI121) [reply tests/test_formato_json.py]: sorry for that
I have updated the tests and also ran black and flake8 before committing as you asked
